addons/web_kanban/static/src/css/kanban.sass

```diff
@@ -279,6 +312,19 @@
     &.oe_kanban_grouped
         .oe_kanban_record
             margin-bottom: 4px
+        .fa 
+            margin-right: 4px
+        .oe_dropdown_kanban
+        &:not(.oe_opened)
+            .fa
+                display: none
+        .oe_dropdown_kanban.oe_opened
+            .fa 
+                display: inline-block
+        &:hover
+            .oe_dropdown_kanban
+                .fa 
+                    display: inline-block
```

Comment:
i suppose you should fix one these rules instead of writing new ones
https://github.com/odoo/odoo/blob/master/addons/web_kanban/static/src/css/kanban.sass#L418